### PR TITLE
Basic encyclopedia page for opening policy slots

### PR DESCRIPTION
**Pull request by MichaelCourtney**
https://github.com/freeorion/freeorion/issues/3528

### CODE REVIEW

File: default/stringtables/en.txt [hunk 1]
```diff
@@ -5338,6 +5338,34 @@ Policies
 ENC_POLICY_DESC
 Improved or situational effects that can be adopted by empires.
 
+ENC_POLICY_SLOTS
+Policy Slots
+
+ENC_POLICY_SLOTS_DESC
+'''Slots that can be used to adopt policies.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]:
```suggestion
'''Policy slots are used to adopt policies on the government screen. Slots and policies have categories, economic, military, and social, and only policies of the appropriate category can be put into a slot.

Slots are limited, and opening more slots is an important aspect of empire development, as it allows adopting more policies than can be placed in the initially available slots. Slots can be opened by owning buildings, from buildings, researching techs, owning ships with certain ship hulls, and by adopting policies. Generally, a given way of opening a slot will only work once per empire, so just owning additional copies of the same building or hull won't give additional slots beyond the first for that building or hull.
```

File: default/stringtables/en.txt [hunk 2]
```diff
@@ -5338,6 +5338,34 @@ Policies
...
+
+Opening Economic Policy Slots:
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]:
```suggestion
Economic Policy Slots:
```

File: default/stringtables/en.txt [hunk 3]
```diff
@@ -5338,6 +5338,34 @@ Policies
...
+[[buildingtype BLD_IMPERIAL_PALACE]]
+[[policy PLC_INTERSTELLAR_INFRA]]
+[[policy PLC_PLANETARY_INFRA]]
+[[policy PLC_SYSTEM_INFRA]]
+[[tech PRO_SENTIENT_AUTOMATION]]
+
+Opening Military Policy Slots:
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]:
```suggestion
Military Policy Slots:
```

File: default/stringtables/en.txt [hunk 4]
```diff
@@ -5338,6 +5338,34 @@ Policies
...
+[[buildingtype BLD_MILITARY_COMMAND]]
+[[shiphull SH_SENTIENT]]
+[[tech LRN_DISTRIB_THOUGHT]]
+[[tech LRN_TIME_MECH]]
+
+Opening Social Policy Slots:
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]:
```suggestion
Social Policy Slots:
```

**Comment by MichaelCourtney** [comment]
Added suggested changes, squished commits, fixed plurals

File: default/stringtables/en.txt [hunk 5]
```diff
@@ -5338,6 +5338,36 @@ Policies
 ENC_POLICY_DESC
 Improved or situational effects that can be adopted by empires.
 
+ENC_POLICY_SLOTS
+Policy Slot
+
+ENC_POLICY_SLOTS_DESC
+'''Policy slots are used to adopt policies on the government screen. Slots and policies have categories, economic, military, and social, and only policies of the appropriate category can be put into a slot.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]:
```suggestion
'''Policy slots are used to adopt policies on the government screen. Slots and policies have categories (economic, military, and social) and only policies of the appropriate category can be put into a slot.
```

File: default/stringtables/en.txt [hunk 6]
```diff
@@ -5338,6 +5338,36 @@ Policies
...
+
+Slots are limited, and opening more slots is an important aspect of empire development, as it allows adopting more policies than can be placed in the initially available slots. Slots can be opened by owning buildings, from buildings, researching techs, owning ships with certain ship hulls, and by adopting policies. Generally, a given way of opening a slot will only work once per empire, so just owning additional copies of the same building or hull won't give additional slots beyond the first for that building or hull.
```
Comment (geoffthemedio) [inline default/stringtables/en.txt]:
```suggestion
Slots are limited, and opening more slots is an important aspect of empire development, as it allows adopting more policies than can be placed in the initially available slots. Slots can be opened by owning buildings, researching techs, owning ships with certain ship hulls, and by adopting policies. Generally, a given way of opening a slot will only work once per empire, so just owning additional copies of the same building or hull won't give additional slots beyond the first for that building or hull.
```

**Comment by MichaelCourtney** [comment]
updated

**Pull request merged**